pipeline.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Minor comment, but I would consider moving ci_script and pipeline.sh to the scripts directory